src/Pages/cart/pages/Items.jsx

```diff
@@ -1,66 +0,0 @@
-import { Button } from "@chakra-ui/react";
-import React from "react";
-import { useContext } from "react";
-import { CartContext } from "../context/CartContext";
-
-const Items = ({ id, title, Quantity, price, img }) => {
```

Comment:
Give a proper name to your components. It should be readable and determining what this component is about